test/parallel/test-zlib-write-after-close.js

```diff
@@ -26,10 +26,12 @@ const zlib = require('zlib');
 zlib.gzip('hello', common.mustCall(function(err, out) {
   const unzip = zlib.createGunzip();
   unzip.close(common.mustCall());
-
-  unzip.write(out);
-  unzip.on('error', common.expectsError({
-    code: 'ERR_STREAM_DESTROYED',
-    type: Error
-  }));
+  common.expectsError(
+    () => unzip.write(out),
+    {
+      code: 'ERR_STREAM_DESTROYED',
+      type: Error,
+      message: 'Cannot call write after a stream was destroyed'
+    }
+  );
 }));
```

Comment:
This updated test is what's causes the failure. Only reverting this would also make the test pass.